main/mySpace/all_attendance_list.php

```diff
@@ -0,0 +1,1185 @@
+<?php
```

Comment:
Are you sure all the code is being used? This looks just like main/mySpace/myStudents.php

Reply:
I use it as a base, but I don't use the skills, courses and tests, using the entire mystudent structure would saturate the view of the data that is relevant. In any case, this view still needs to be detailed on how the data would be structured.

![image](https://user-images.githubusercontent.com/18097392/87440831-0793c300-c5b8-11ea-94e6-81dcc20b3af3.png)
These data would have no relevance in view.

![image](https://user-images.githubusercontent.com/18097392/87441011-390c8e80-c5b8-11ea-99d2-5346947950c0.png)
Only interested in this section at the moment.


In the same way it is still in process.

Reply:
It's better to create a function that returns that part in yellow, and then call that function in main/mySpace/myStudents.php

Or only show that yellow part in all_attendance_list.php

Reply:
It's not a good idea to duplicate main/mySpace/myStudents.php